static/index.html

```diff
@@ -95,6 +96,23 @@ <h3>Optional encryption key</h3>(encrypt your message before sending it to the
     </div>
 </body>
 
+<div class="footer">
```

Comment:
I think this whole thing should be inside of a `<div class="container">`

Reply:
makes sense